.github/workflows/ci.yaml

```diff
@@ -25,11 +25,13 @@ jobs:
       - name: Install Go
         uses: actions/setup-go@v2
         with:
-          go-version: "1.14.6"
+          go-version: "1.17"
       - name: Generate
         run: make generate
+      - name: go clean -testcache
+        run: go clean -testcache
```

Comment:
Do we really need this? If so why?

Reply:
sure, It was in the Makefile, so I added for consistency

Reply:
ok makes sense. consistency > sense :D